server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -71,11 +72,18 @@ object InteractiveSession extends Logging {
       mockClient: Option[RSCClient] = None): InteractiveSession = {
     val appTag = s"livy-session-$id-${Random.alphanumeric.take(8).mkString}"
 
+    val livyConfWithTestConf = if (request.conf.contains(RSCConf.Entry.TEST_PING_JOB_ERROR.key())) {
+      new LivyConf().setAll(livyConf).set(RSCConf.Entry.TEST_PING_JOB_ERROR.key(), "true")
```

Comment:
What is the purpose of using the conf, I'm not sure as a user how to use it, what it stands for?